plugins/mcreator-localization/lang/texts.properties

```diff
@@ -828,8 +828,8 @@ blockly.block.compare_mcitems=%3 %1 = %4 %2
 blockly.block.compare_mcitems_oredictionary=Is %1 tagged in item tags as %2
 blockly.block.console_log=Log %1 with log level %2
 blockly.block.copy_mcitem=Make a copy of %2 %1
-blockly.block.deal_damage=Deal %1 damage to %2 type:
-blockly.block.damagesource_isequalto=Is provided damage source of type %1
+blockly.block.deal_damage=Deal %1 damage to %2 type %3
```

Comment:
I'd change the text to `Deal %1 damage to %2 from source %3`

Reply:
Done